### PR TITLE
Add 3D print files for Simple Duck Case

### PR DESCRIPTION
**Pull request by TimoWielink**
Add .STL files to 3d Print your own Duck Case

### CODE REVIEW

File: 3D print Files/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## ClusterDuck Protocol - Simple Duck Case
+This folder includs two files to 3D print your own Duck Case. This case is designed to be used with the following products.
+
+#### - Helltec Wifi Lora ESP32 https://heltec.org/project/wifi-lora-32/
+Note: Check Local Rules and regulations for frequency. Can be bought from different sellers. 
+
+#### - Batterie Case https://tinyurl.com/qsep8ru
```
Comment (krook) [inline 3D print Files/README.md]: Typo on "Battery Case"

**Comment by krook** [comment]
How's this look @knouse1344 and @nfeuer?

**Comment by knouse1344** [comment]
Looks fantastic, thanks @TimoWielink

File: 3D print Files/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## ClusterDuck Protocol - Simple Duck Case
+This folder includs two files to 3D print your own Duck Case. This case is designed to be used with the following products.
+
+#### - Helltec Wifi Lora ESP32 https://heltec.org/project/wifi-lora-32/
```
Comment (Salzi) [inline 3D print Files/README.md]: Typo: Heltec

Comment (knouse1344) [reply 3D print Files/README.md]: thank you!

**Pull request merged**